apps/apputil.hpp

```diff
@@ -221,9 +247,16 @@ inline options_t ProcessOptions(char* const* argv, int argc, std::vector<OptionS
     {
         const char* a = *p;
         // cout << "*D ARG: '" << a << "'\n";
-        if ( moreoptions && a[0] == '-' )
+        if (moreoptions && a[0] == '-')
         {
-            current_key = a+1;
+            string key(a + 1);  // omit '-'
+            size_t pos = key.find_first_of(":=");
+            if (pos == string::npos)
+                pos = key.find(' ');
+            string value = pos == string::npos ? "" : key.substr(pos + 1);
+            key = key.substr(0, pos);
+
+            current_key = key;
```

Comment:
@ethouris Added handling of `-option:value` syntax. To review.

Reply:
Rest of the things looks good.